packages/lib/services/interop/InteropService_Exporter_Md.ts

```diff
@@ -124,15 +131,36 @@ export default class InteropService_Exporter_Md extends InteropService_Exporter_
 
 			const noteBody = await this.relaceLinkedItemIdsByRelativePaths_(item);
 			const modNote = Object.assign({}, item, { body: noteBody });
-			const noteContent = await Note.serializeForEdit(modNote);
+			const noteContent = await this.getNoteExportContent_(modNote);
 			await shim.fsDriver().mkdir(dirname(noteFilePath));
 			await shim.fsDriver().writeFile(noteFilePath, noteContent, 'utf-8');
 		}
 	}
 
+	async findReasonableFilename(_resource: any, filePath: string) {
```

Comment:
All new class methods should set the member accessibility. It's only a warning in eslint because it would be too much work to update all the methods from the old code, but for new ones we should always set them.

Reply:
Thanks, I saw the warnings, but it was late when I pushed and I didn't think anything of it.